pypeclub/OpenPype
# Maya: review viewport settings

# Description

Publish resolution setting were override while capture .

# Bug Details
maintain_aspect_ratio was True by default while no calculation on it in code or getting data from UI .. so I made it False when Width and Height were given in publish setting.

# Testing Notes

1. Set resolution in studio setting >> project >> Maya >> Publish  >> Extract Playblast .
5. Publish `Review`.
6. Check resolution for Review and Thumbnail.
